packages/job/src/JobControl.tsx

```diff
@@ -141,7 +141,7 @@ export const JobControl = (props: {
     setLoading(true)
     try {
       await dmtApi.removeJob({ jobUid: jobUID })
-      await dmssApi.documentRemove({
+      await dmssApi.explorerRemove({
```

Comment:
```suggestion
      await dmssApi.explorerRemove({
```
`explorer` has been merged into `document`, no?

Reply:
fixed